api/config/config_local.json

```diff
@@ -31,7 +31,8 @@
     "freeTierCostAlertThresholds": [
       0.5,
       0.75
-    ]
+    ],
+    "carahsoftEmail": ""
```

Comment:
Will the email still go somewhere so we can view it? e.g. just to the support list?

Reply:
Can we use support@researchallofus.org as email receipt for lower env? If yes, that would be great

Reply:
We shouldn't send lower env emails here, may be very spammy and this is the production list. I thought we had a fake support email address but I can't find it. Would be ideal if it just goes to a listserv we control - but I don't see one right now.

Fine to just leave it blank then

Reply:
Sorry I sent this as two batches so you might have missed this - please revert this